### ISSUE TITLE
Forms without lables

### ISSUE THREAD

**Issue by Horlix** [canonical/ubuntu.com#11884]
These include:
- https://ubuntu.com/download/desktop/thank-you
- https://ubuntu.com/openstack/install
- https://ubuntu.com/robotics
- https://ubuntu.com/robotics/community

All forms need a full review, I advise a patter library review to ensure all elements are reviewed.

**Comment by Horlix** [comment]
Full list if URLs

[HTML form control has no accessible name.csv](https://github.com/canonical-web-and-design/ubuntu.com/files/9213799/HTML.form.control.has.no.accessible.name.csv)

**Comment by Onibenjo** [comment]
These forms have recently been redesigned and some use a centralised form library we have which improved the accessibility of our forms. We are in a phase of replacing all forms across the site with the library.

Thanks for opening.
